### PR TITLE
[search][tmp] Show address in PP. Need to recheck PP layout for the production.

### PR DESCRIPTION
**Pull request by vng**

### CODE REVIEW

**Comment by ygorshenin** [comment]
LGTM

File: map/address_finder.cpp [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #include "map/framework.hpp"
 
 #include "search/categories_holder.hpp"
+#include "search/reverse_geocoder.hpp"
```
Comment (mpimenov) [inline map/address_finder.cpp]: sort

**Comment by mpimenov** [comment]
LGTM

**Comment by vng** [comment]
PTAL

**Comment by mpimenov** [comment]
LGTM

**Pull request merged**